kotest-assertions/kotest-assertions-json/src/commonTest/kotlin/io.kotest.assertions.json/schema/ObjectSchemaTest.kt

```diff
@@ -3,12 +3,11 @@ package io.kotest.assertions.json.schema
 import io.kotest.assertions.shouldFail
 import io.kotest.core.spec.style.FunSpec
 import io.kotest.matchers.shouldBe
-import org.intellij.lang.annotations.Language
 
 
 class ObjectSchemaTest : FunSpec(
    {
-      fun json(@Language("JSON") raw: String) = raw
+      fun json(raw: String) = raw
```

Comment:
likewise

Reply:
Good catch - I've fixed this in https://github.com/kotest/kotest/pull/3021/commits/c1dbc548621d9f2dc7cfac13fece6275fd08ae0d.